### PR TITLE
GitHub API

### PR DESCRIPTION
**Pull request by KentoNishi**
Added GitHub API stuff.

### CODE REVIEW

File: autoencoder/train.py [hunk 1]
```diff
@@ -1,78 +1,78 @@
...
+    plt.gray()
+    ax.get_xaxis().set_visible(False)
+    ax.get_yaxis().set_visible(False)
+    
+    # display reconstruction
+    ax = plt.subplot(3, n, i + 1 + 2*n)
+    plt.imshow(decoded_imgs[i].reshape(28, 28))
+    plt.gray()
+    ax.get_xaxis().set_visible(False)
+    ax.get_yaxis().set_visible(False)
```
Comment (lithomas1) [inline autoencoder/train.py]: we are using tensorboard images in order to display images. matplotllib should be removed as dependency

File: color-autoencoder/run.py [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
-from keras.models import load_model
-from keras.layers import Dense, Dropout, Activation, Flatten, Conv2D, MaxPooling2D, UpSampling2D, Input
-from keras.datasets import mnist
-from keras.models import load_model
-from keras.utils import CustomObjectScope
-from keras.initializers import glorot_uniform
-
-with CustomObjectScope({'GlorotUniform': glorot_uniform()}):
-        model = load_model('autoencoder.h5')
+import pandas as pd
```
Comment (lithomas1) [inline color-autoencoder/run.py]: remove unused imports

File: autoencoder/train.py (hunk 1 above)
Comment (KentoNishi) [reply autoencoder/train.py]: This is from the original code. I only added the GitHub API stuff. Change it on another branch, or clone and fix this one. I'm too lazy

**Comment by KentoNishi** [comment]
I'll add comments to the js

**Comment by lithomas1** [comment]
> 
> 
> I'll add comments to the js

can you review my imageops.py code? also, my code with have to merge into master before yours. We might have some nasty merge conflicts because we are all editing train.py

File: autoencoder/train.py (hunk 1 above)
Comment (lithomas1) [reply autoencoder/train.py]: changed on my branch but not yet working. please look at it

**Comment by lithomas1** [comment]
don't merge until my branch is merged and don't worry about status checks

**Comment by KentoNishi** [comment]
Can you merge the other one then

**Comment by lithomas1** [comment]
> 
> 
> Can you merge the other one then

review plz. and also devops is being a pain again bc no build cache so it takes forever to download everything again

**Comment by KentoNishi** [comment]
I did it

**Pull request merged**